tests/integration/components/fm-widgets/select-test.js

```diff
@@ -145,7 +145,7 @@ module('Integration | Component | fm-widget:select', function (hooks) {
   });
 
   test('supports undefined content in widgetAttrs', async function (assert) {
-    await render(hbs`{{fm-widgets/select value=this.value widgetAttrs=(hash content=this.content optionValueLabel='label' optionValuePath='value')}}`);
+    await render(hbs`{{fm-widgets/select value=this.value widgetAttrs=(hash content=content optionValueLabel='label' optionValuePath='value')}}`);
```

Comment:
I could be wrong, but I think this might throw a deprecation warning for the not having the `this` keyword before `content`. 

This is a test so I'm not 100% sure if it even matters here.

Reply:
Ah, I think you might be correct. Good catch 👍. We should be using `this` in modern Ember.